### PR TITLE
Add OARS data

### LINKED ISSUE #625: Add OARS to appstream file
Builds are failing because OARS is missing from appstream.

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #625

### CODE REVIEW

File: data/io.elementary.appcenter.appdata.xml.in [hunk 1]
```diff
@@ -147,6 +147,35 @@
...
+        <content_attribute id="sex-appearance">none</content_attribute>
+        <content_attribute id="language-profanity">none</content_attribute>
+        <content_attribute id="language-humor">none</content_attribute>
+        <content_attribute id="language-discrimination">none</content_attribute>
+        <content_attribute id="social-chat">none</content_attribute>
+        <content_attribute id="social-info">none</content_attribute>
+        <content_attribute id="social-audio">none</content_attribute>
+        <content_attribute id="social-location">none</content_attribute>
+        <content_attribute id="social-contacts">none</content_attribute>
+        <content_attribute id="money-purchasing">none</content_attribute>
```
Comment (cassidyjames) [inline data/io.elementary.appcenter.appdata.xml.in]: umm?

File: data/io.elementary.appcenter.appdata.xml.in [hunk 2]
```diff
@@ -147,6 +147,35 @@
...
+        <content_attribute id="sex-themes">none</content_attribute>
+        <content_attribute id="sex-homosexuality">none</content_attribute>
+        <content_attribute id="sex-prostitution">none</content_attribute>
+        <content_attribute id="sex-adultery">none</content_attribute>
+        <content_attribute id="sex-appearance">none</content_attribute>
+        <content_attribute id="language-profanity">none</content_attribute>
+        <content_attribute id="language-humor">none</content_attribute>
+        <content_attribute id="language-discrimination">none</content_attribute>
+        <content_attribute id="social-chat">none</content_attribute>
+        <content_attribute id="social-info">mild</content_attribute>
```
Comment (cassidyjames) [inline data/io.elementary.appcenter.appdata.xml.in]: This is caused by this option in OARS:

># Information Sharing
>_Defined as sharing information with a legal entiry typically used for advertising or for sending back diagnostic data._

I chose *Checking for the latest application version*

File: data/io.elementary.appcenter.appdata.xml.in [hunk 3]
```diff
@@ -147,6 +147,35 @@
...
+        <content_attribute id="social-audio">none</content_attribute>
+        <content_attribute id="social-location">none</content_attribute>
+        <content_attribute id="social-contacts">none</content_attribute>
+        <content_attribute id="money-purchasing">intense</content_attribute>
```
Comment (cassidyjames) [inline data/io.elementary.appcenter.appdata.xml.in]: Caused by:

># In-App Purchases
>_Defined as items or points that a user can buy for use within a virtual world to improve a character or enhance the playing experience._

I chose **Ability to spend real money in-app**

**Comment by danrabbit** [comment]
Yeah I misread the descriptions. It seemed like it was asking more about things that would apply to games and social apps. I'm not sure if "Advertising" applies here based on the wording on the site. It seems like that's talking about advertising for real world products or brands. I guess that's hard to define since what we're doing on the homepage is the real-life equivalent of an end-cap in the grocery store. I guess that's technically advertising but it doesn't feel like what we mean when we say advertising as in when a game has ads in it

**Comment by cassidyjames** [comment]
@danrabbit I checked, and changing "Advertising" had no effect on the ratings. I presume advertising would mark `money-purchasing` as mild, but since we straight up process payments, it's moot.

**Pull request merged**